state/grants.go

```diff
@@ -1389,6 +1391,69 @@ func GetHardCodedGrants() []HardGrant {
 			HardGrant{grant2020_04, 20000e8, validateAddress("FA2rrwFVvkFYwyGFHVBMwRqTpycuZiagrQdcbPWzuoEwJQxjDwi3")},
 			// --------------------------------------------------------
 
+			// ********************************
+			// **** Grant Round 2021-01 ****
+			// ********************************
+
+			// --------------------------------------------------------
+			// Anton Ilzheev-The Factoid Authority-De Facto -- 3125 FCT
+			// Factom Open Node 2021-01-25 - 2021-02-16
+
+			// The Factoid Authority total: 625 FCT
+			HardGrant{grant2021_01, 625e8, validateAddress("FA2LV4s7LKA9BTgWaJNvcr9Yq8rpiH2XD3vEPY3nwSiNSrnRgkpK")},
+
+			// De Facto total: 2500 FCT
+			HardGrant{grant2021_01, 2500e8, validateAddress("FA3X3d3ceDYD7v3aBCeEBcvvgiTZNeiFkN8jJ897EbMuBzHt6xhP")},
+			// --------------------------------------------------------
+
+			// --------------------------------------------------------
+			// Anton Ilzheev-Website Committee -- 1675 FCT
+			// Website Committee grant 2021-01-25 - 2021-02-16
+
+			// Website Committee total: 1675 FCT
+			HardGrant{grant2021_01, 1675e8, validateAddress("FA3TG1mi25XfJSVeNT5vS9uLi8VQxjK8ju9pGCp8Po6rrk5RXPLH")},
+			// --------------------------------------------------------
+
+			// --------------------------------------------------------
+			// PaulSnow-Anton Ilzheev-De Facto -- 900 FCT
+			// Factom Oracle Master 2021-01-25 - 2021-02-16
+
+			// PaulSnow total: 450 FCT
+			HardGrant{grant2021_01, 450e8, validateAddress("FA29PLacx8jVBMZoQrjgES5RBsfooMsWHG5Ep8uUhVmeesGFY4xp")},
+
+			// De Facto total: 450 FCT
+			HardGrant{grant2021_01, 450e8, validateAddress("FA3X3d3ceDYD7v3aBCeEBcvvgiTZNeiFkN8jJ897EbMuBzHt6xhP")},
+			// --------------------------------------------------------
+
+			// --------------------------------------------------------
+			// WB -- 50000 FCT
+			// Initial FCT for Council/Director 2021-01-25 - 2021-02-16
+
+			// WB total: 50000 FCT
+			HardGrant{grant2021_01, 50000e8, validateAddress("FA3m63HmmkqqtUwxsxPRZLdoq2JJgU2LFC92hcM5jnauVAnYE2W6")},
+			// --------------------------------------------------------
+
+			// --------------------------------------------------------
+			// Anton Ilzheev-De Facto -- 6500 FCT
+			// WFCT gateway development backpay 2021-01-25 - 2021-02-16
+
+			// De Facto total: 6500 FCT
+			HardGrant{grant2021_01, 6500e8, validateAddress("FA3X3d3ceDYD7v3aBCeEBcvvgiTZNeiFkN8jJ897EbMuBzHt6xhP")},
+			// --------------------------------------------------------
+
+			// --------------------------------------------------------
+			// PaulSnow-Factom Inc-De Facto-Core Committee -- 43000 FCT
+			// Anchor Rework 2021-01-25 - 2021-02-16
+
```

Comment:
`// Per agreement during question of the grant, the FCT price should be adjusted at the rate`
`// specified by the 144 block average reported by the EC Rate Bot on discord ( #ec-rate-alert )`
`// at a date prior to deploying the grant.  On 2021-2-17 , this rate was $2.57.  This changes the` 
`// tokens issued from 43000 to 23434, or 11712 FCT to Factom Inc. and De Facto`

Reply:
Thanks @PaulSnow . The math looks slightly off, assuming a new total of 23434, and the core committee part remaining 1000, it gives me: (23434-1000)/2=11217, which is different from your number. It looks like the last 3 digits were suffled, so maybe it's a typo?